pom.xml

```diff
@@ -218,26 +199,23 @@
     </dependencies>
     <dependencyManagement>
         <dependencies>
+            <dependency>
+                <groupId>io.jenkins.tools.bom</groupId>
+                <artifactId>bom-2.150.x</artifactId>
+                <version>5</version>
+                <scope>import</scope>
+                <type>pom</type>
+            </dependency>
             <dependency>
                 <groupId>org.jenkins-ci</groupId>
                 <artifactId>annotation-indexer</artifactId>
```

Comment:
Note: probably superseded by use of core BOM in 4.x parent. @jtnord could confirm

Reply:
I don't know what this means.

Reply:
That you would pick up https://github.com/jenkinsci/jenkins/blob/c376ffe0a41ff4a2100d41ca679a2c7995557dfd/bom/pom.xml#L168-L172 if you updated to the beta parent and got https://github.com/jenkinsci/plugin-pom/pull/269. Can ignore for now.

Reply:
should not be required in latest parent pom